### ISSUE TITLE
Reenable Py3 tests in test.test_loopback

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6136]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#6136|
|Type|enhancement|
|Created|2012-10-16 15:20:00Z|
|Branch|https://github.com/twisted/twisted/tree/loopback-py3-6136|

`IReactorUNIX` is implemented and works, so now update a test that incorrectly skips.

<details><summary>Searchable metadata</summary>

```
trac-id__6136 6136
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_loopback_py3_6136 branches/loopback-py3-6136
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1350400800000000 1350400800000000
changetime__1435844180983092 1435844180983092
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

`twisted.test.test_loopback` has some tests skipped that should be unskipped when this is resolved.

**Comment by twisted-trac** [comment]
|<img alt="nott_'s avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|nott_ set owner to nott_|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl set owner to @hawkowl|
|-|-|

This has been fixed in trunk but the tests haven't been unskipped. Lemme fix that.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)|@hawkowl removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl commented|
|-|-|

Builders spun, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_7"></a>|@adiroiban set owner to @hawkowl|
|-|-|

twisted.test.test_process.PosixProcessPTYTests.test_executionError was failing... I have re-run it and it is green.

please merge. thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45178]) Merge loopback-py3-6136: Re-enable UNIX tests in test_loopback on Py3

Author: hawkowl
Reviewer: adiroiban
Fixes: [#6136](https://github.com/twisted/twisted/issues/6136)
